kotlin-native/build-tools/src/main/kotlin/org/jetbrains/kotlin/RunKotlinNativeTask.kt

```diff
@@ -59,22 +59,22 @@ open class RunKotlinNativeTask @Inject constructor(private val linkTask: Task,
         val useCset = project.findProperty("useCset")?.toString()?.toBoolean() ?: false
         project.exec {
             if (useCset) {
-                it.executable = "cset"
-                it.args("shield", "--exec", "--", executable)
+                executable = "cset"
+                args("shield", "--exec", "--", executable)
```

Comment:
Isn't this `executable` gets `"cset"` here? It should set `RunKotlinNativeTask`'s `executable` property. I wounder also why `it` changed to `this`.

Reply:
this is because of `kotli-sam-with-receiver` plugin